test-cases/bentsi.yaml

```diff
@@ -0,0 +1,32 @@
+test_duration: 10
```

Comment:
Is this file here accidentally?

Reply:
I will remove it before the merge. Used to test the PR on by job....